.storybook/main.js

```diff
@@ -6,7 +6,6 @@ const storybookConfig = storybookDefaultConfig( {
 	stories: [
 		'../client/**/*.stories.{js,jsx,ts,tsx}',
 		'../packages/design-picker/src/**/*.stories.{ts,tsx}',
-		'../packages/components/src/**/*.stories.{js,jsx,ts,tsx}',
```

Comment:
Have we considered if it's possible to have both proper isolation and a single Storybook instance with all Calypso stories in it?

Reply:
It is definitely possible. I built the stylesheet isolation structure in the Gutenberg Storybook using lazy-loaded stylesheets that can be loaded and unloaded at will. I'll probably come back to this once I'm done with `@automattic/components` and have bandwidth to tinker with `client/components`.